### PR TITLE
Asynchronous job

### PR DESCRIPTION
**Pull request by JesseLabruyere**
Added asynchronous jobs using Beanstalkd queueing for storing: measurements, devices, sensors.

### CODE REVIEW

**Review by markdingemanse — CHANGES_REQUESTED** [review]
$request->all() moet consistent worden toegepast

File: app/Http/Controllers/Measurements/MeasurementController.php [hunk 1]
```diff
@@ -37,8 +38,11 @@ public function show() {
         //... TODO
     }
 
-    public function store() {
-        //... TODO
+    public function store(MeasurementStoreRequest $request) {
+        $requestArray = (array) $request;
```
Comment (markdingemanse) [inline app/Http/Controllers/Measurements/MeasurementController.php]: hierzo dit mag dus niet

**Review by markdingemanse — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Measurements/MeasurementController.php (hunk 1 above)
Comment (JesseLabruyere) [reply app/Http/Controllers/Measurements/MeasurementController.php]: Dit is al gefixt in de laatste commit.

**Review by markdingemanse — APPROVED** [review]

File: app/Http/Controllers/Measurements/MeasurementController.php (hunk 1 above)
Comment (JesseLabruyere) [reply app/Http/Controllers/Measurements/MeasurementController.php]: https://github.com/lorawan-team/Delta_LoRaWAN/pull/22/commits/eb411bfe0eb269625422df4ce9e55f878920dbf6

**Pull request merged**